src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.scss

```diff
@@ -3,3 +3,11 @@
   flex-direction: column;
   justify-content: space-evenly;
 }
+
+.removeBorder {
+  border: 0;
+}
+
+.hello {
+  align-items: center !important;
+}
```

Comment:
World? This style is not even used here. Aren't we running a css linter to detect unused styles?